files/master-bullseye/marvell_x86.patch

```diff
@@ -0,0 +1,8537 @@
++$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(FALCON_DB98CX8540_16CD_PLATFORM)
++$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(FALCON_DB98CX8580_32CD_PLATFORM)
++$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(FALCON_DB98CX8514_10CC_PLATFORM)
+ ifeq ($(INSTALL_DEBUG_TOOLS),y)
+ $(SONIC_ONE_IMAGE)_DOCKERS += $(SONIC_INSTALL_DOCKER_DBG_IMAGES)
+ $(SONIC_ONE_IMAGE)_DOCKERS += $(filter-out $(patsubst %-$(DBG_IMAGE_MARK).gz,%.gz, $(SONIC_INSTALL_DOCKER_DBG_IMAGES)), $(SONIC_INSTALL_DOCKER_IMAGES))
+diff --git a/platform/marvell/platform-db98cx8514-10cc.mk b/platform/marvell/platform-db98cx8514-10cc.mk
+new file mode 100644
+index 0000000..4a1435f
+--- /dev/null
++++ b/platform/marvell/platform-db98cx8514-10cc.mk
+@@ -0,0 +1,12 @@
++# FALCON Platform 
++
++FALCON_VERSION = 1.0
++export FALCON_VERSION
++
++FALCON_DB98CX8514_10CC_PLATFORM = sonic-platform-db98cx8514-10cc-db98cx8514_$(FALCON_VERSION)_$(CONFIGURED_ARCH).deb
++$(FALCON_DB98CX8514_10CC_PLATFORM)_SRC_PATH = $(PLATFORM_PATH)/sonic-platform-db98cx8514-10cc
++$(FALCON_DB98CX8514_10CC_PLATFORM)_DEPENDS += $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
++$(FALCON_DB98CX8514_10CC_PLATFORM)_PLATFORM = x86_64-marvell_db98cx8514_10cc-r0 
++
++SONIC_DPKG_DEBS += $(FALCON_DB98CX8514_10CC_PLATFORM)
++
+diff --git a/platform/marvell/platform-db98cx8540-16cd.mk b/platform/marvell/platform-db98cx8540-16cd.mk
+new file mode 100644
+index 0000000..a60d662
+--- /dev/null
++++ b/platform/marvell/platform-db98cx8540-16cd.mk
+@@ -0,0 +1,12 @@
++# FALCON Platform 
++
++FALCON_VERSION = 1.0
++export FALCON_VERSION
++
++FALCON_DB98CX8540_16CD_PLATFORM = sonic-platform-db98cx8540-16cd-db98cx8540_$(FALCON_VERSION)_$(CONFIGURED_ARCH).deb
++$(FALCON_DB98CX8540_16CD_PLATFORM)_SRC_PATH = $(PLATFORM_PATH)/sonic-platform-db98cx8540-16cd
++$(FALCON_DB98CX8540_16CD_PLATFORM)_DEPENDS += $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
++$(FALCON_DB98CX8540_16CD_PLATFORM)_PLATFORM = x86_64-marvell_db98cx8540_16cd-r0 
++
++SONIC_DPKG_DEBS += $(FALCON_DB98CX8540_16CD_PLATFORM)
++
+diff --git a/platform/marvell/platform-db98cx8580-32cd.mk b/platform/marvell/platform-db98cx8580-32cd.mk
+new file mode 100644
+index 0000000..f1f9210
+--- /dev/null
++++ b/platform/marvell/platform-db98cx8580-32cd.mk
+@@ -0,0 +1,12 @@
++# FALCON Platform 
++
++FALCON_VERSION = 1.0
++export FALCON_VERSION
++
++FALCON_DB98CX8580_32CD_PLATFORM = sonic-platform-db98cx8580-32cd-db98cx8580_$(FALCON_VERSION)_$(CONFIGURED_ARCH).deb
++$(FALCON_DB98CX8580_32CD_PLATFORM)_SRC_PATH = $(PLATFORM_PATH)/sonic-platform-db98cx8580-32cd
++$(FALCON_DB98CX8580_32CD_PLATFORM)_DEPENDS += $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
++$(FALCON_DB98CX8580_32CD_PLATFORM)_PLATFORM = x86_64-marvell_db98cx8580_32cd-r0 
++
++SONIC_DPKG_DEBS += $(FALCON_DB98CX8580_32CD_PLATFORM)
++
+diff --git a/platform/marvell/rules.mk b/platform/marvell/rules.mk
+index da23e53..6bf0ade 100644
+--- a/platform/marvell/rules.mk
++++ b/platform/marvell/rules.mk
+@@ -3,6 +3,10 @@ include $(PLATFORM_PATH)/docker-syncd-mrvl.mk
```

Comment:
Pls align all the changes similar to marvell-armhf platform.
Also docker-syncd-mrvl.mk should map to bullseye makefile.